rundeck/map.jinja

```diff
@@ -14,12 +14,18 @@
         'rundeckd_path': '/etc/default/rundeckd',
         'login_path': '/etc/rundeck/jaas-loginmodule.conf',
         'pkg_java': salt['grains.filter_by']({
-            '14': 'openjdk-7-jdk',
-            '16': 'openjdk-8-jdk',
-        }, grain='osmajorrelease'),
+            'Ubuntu': salt['grains.filter_by']({
+                '14': 'openjdk-7-jdk',
+                '16': 'openjdk-8-jdk',
```

Comment:
As this is being changed, it would be a good time to have a `default` value as well, like is done [here](https://github.com/saltstack-formulas/ntp-formula/blob/5049d172cfab21c0b879ab843c11f49e7a84bacc/ntp/map.jinja#L44).